fedimint-api/src/core/server.rs

```diff
@@ -183,8 +198,16 @@ where
     }
 
     /// Blocks until a new `consensus_proposal` is available.
-    async fn await_consensus_proposal(&self, dbtx: &mut DatabaseTransaction<'_>) {
```

Comment:
This should already be prefixed here.

Reply:
This should be fixed. No longer wrapping the transaction in server.rs